docker/single-process/docker-compose.yml

```diff
@@ -1,45 +1,48 @@
-# This needs at least compose 1.6.0
-version: '2'
+version: '3'
 
 services:
-  mysqldata:
```

Comment:
Not sure about this change, if someone is currently using this compose file and updates to this version mysql would start with an empty database.

Are there benefits in using volumes instead of "volume containers"?

Reply:
I suppose people will have to export data -- is there a release schedule to pace this kind of changes? I imagine it would make sense to eventually abandon the rather hacky and legacy data-only container setup. See here for some [discussions](https://github.com/moby/moby/issues/17798#issuecomment-154815207).

Reply:
The problem is we don't have a release cycle, we try to not introduce breaking changes as good as we can because master is always supposed to work the way it did before.